platform/use-platform/virtual-clusters/add-host-cluster.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Add existing host clusters
+sidebar_label: Add Existing Host Clusters
+sidebar_position: 3
+description: Add externally deployed host clusters to Platform with visibility-only or full management capabilities
+---
+
+import PageVariables from '@site/src/components/PageVariables';
+import InterpolatedCodeBlock from '@site/src/components/InterpolatedCodeBlock';
+
+You can register existing host clusters to an installed vCluster Platform running in another host cluster.  
+This enables centralized management and visibility of virtual clusters running across multiple host clusters from a single Platform deployment.
+
+### Register host cluster with CLI
```

Comment:
This should use the <Step> and <Flow> components to make it visually consistent with other pagers.
When referencing elements from the platform UI we use this style of reference ([example here](https://www.vcluster.com/docs/platform/configure/advanced/audit)).

It is also helpful to create tabs for various installation methods, UI or CLI unless they belong the the same flow. Example here:

<img width="1112" height="526" alt="Image" src="https://github.com/user-attachments/assets/c138c076-b76d-4c23-a1d2-f624d62d4279" />